extensions/amp-story-panning-media/0.1/amp-story-panning-media.js

```diff
@@ -79,7 +80,10 @@ export class AmpStoryPanningMedia extends AMP.BaseElement {
     this.isOnActivePage_ = false;
 
     /** @private {?number} Distance from active page. */
-    this.distance_ = null;
+    this.pageDistance_ = null;
+
+    /** @private {number} Min distance from active page to animate. */
+    this.minPageDistance_ = 2;
```

Comment:
The variable name could inform better that it's used for animation, eg: `maxDistanceToAnimate_`. Also AFAIK this is an upper bound (therefore a max) instead of a min. And in this case, it could make sense to have the comparison be <= instead of < so it's an inclusive bound (moving the values to 0 and 1), meaning a distance equal to this value will animate.